### DISCUSSION TITLE
AV Issue

### DISCUSSION THREAD

**Discussion by AlexYoung28** [HotCakeX/Harden-Windows-Security#285] [Q&A]
Has anyone had experience of

![image](https://github.com/HotCakeX/Harden-Windows-Security/assets/6728021/840f387e-44b2-4cba-b967-5daf917f207d)

I've got Bitdefender on here, but i'm just wondering if it's something i can put an exclusion in just to run to get the results.

**Comment by HotCakeX** [comment]
Hi,
so it's already covered in the [FAQs](https://github.com/HotCakeX/Harden-Windows-Security/wiki/Answers-to-the-Basic-Frequently-Asked-Questions#do-i-need-to-install-a-3rd-party-antivirus-or-security-software), but the error looks like your system has problem with a built-in cmdlet which is `Get-MpPreference` used to get the details of the Microsoft Defender.

God knows what Bitdefender did to the system, maybe try restarting and try again, if it still doesn't work try uninstalling it . The result of the compliance checking won't be much anyway if you didn't use the module to harden the OS, as it fetches the system registry/group policies to audit the system.
